Ginger/GingerCoreCommon/Repository/BusinessFlowLib/ActivitiesGroup.cs

```diff
@@ -204,49 +204,143 @@ public override string ItemName
             }
         }
 
-        public override void UpdateInstance(RepositoryItemBase instance, string partToUpdate, RepositoryItemBase hostItem = null)
+        public override void UpdateInstance(RepositoryItemBase instance, string partToUpdate, RepositoryItemBase hostItem = null, object extraDetails = null)
         {
             ActivitiesGroup activitiesGroupInstance = (ActivitiesGroup)instance;
 
             //Create new instance of source
             ActivitiesGroup newInstance = (ActivitiesGroup)this.CreateInstance();
             newInstance.IsSharedRepositoryInstance = true;
 
+            //newInstance.ActivitiesIdentifiers = this.ActivitiesIdentifiers;
+
             //update required part
             ActivitiesGroup.eItemParts ePartToUpdate = (ActivitiesGroup.eItemParts)Enum.Parse(typeof(ActivitiesGroup.eItemParts), partToUpdate);
-            switch (ePartToUpdate)
+            //newInstance.Guid = activitiesGroupInstance.Guid;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4582993)